test/libsolidity/semanticTests/abiEncoderV2/calldata_array.sol

```diff
@@ -15,9 +15,15 @@ contract C {
     }
 }
 // ====
-// compileViaYul: also
 // EVMVersion: >homestead
+// compileViaYul: also
 // ----
 // f(uint256[][1]): 32, 32, 0 -> true
+// gas ir: 105308
 // f(uint256[][1]): 32, 32, 1, 42 -> true
+// gas ir: 139370
 // f(uint256[][1]): 32, 32, 8, 421, 422, 423, 424, 425, 426, 427, 428 -> true
+// gas ir: 377825
+// gas legacy: 144300
+// gas legacyOptimized: 124231
+// gas irOptimized: 227663
```

Comment:
Yes it is actually. Do you think it should do it differently?

Reply:
I'm not sure. Both are valid ways to do this and to be honest I'm not sure which one is better. I commented mostly because I expected them all to be printed.

I think we need more opinions here.

Reply:
Let's wait for more feedback before updating then :-)

Reply:
I think it makes sense as it is now. In addition to that: I would leave out `gas ir` always except if it is given explicitly. Codegen via IR is meant to be run with the optimizer, so the number should always be ignored unless maybe if we have a test that wants to check if a certain construct is optimized away.